mantle/kola/harness.go

```diff
@@ -621,10 +621,10 @@ func filterTests(tests map[string]*register.Test, patterns []string, pltfrm stri
 }
 
 func filterDenylistedTests(tests map[string]*register.Test) (map[string]*register.Test, error) {
-	var denylisted bool
 	r := make(map[string]*register.Test)
 	for name, t := range tests {
-		// Drop anything which is denylisted directly or by pattern
+		denylisted := false
```

Comment:
Good catch! The original code was running `var denylisted bool` each loop iteration. I missed that in the original PR.

Reply:
Yep. Copy and paste got me in trouble. I didn't pay enough attention to detail and my local tests looked good, but they weren't comprehensive enough.